src/System.Diagnostics.EventLog/src/System/Diagnostics/NativeMethods.cs

```diff
@@ -2,22 +2,11 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Runtime.InteropServices;
-
 namespace Microsoft.Win32
 {
     internal static class NativeMethods
     {
-        public readonly static HandleRef NullHandleRef = new HandleRef(null, IntPtr.Zero);
-
-        public const int WAIT_OBJECT_0 = 0x00000000;
-        public const int WAIT_TIMEOUT = 0x00000102;
-        public const int WAIT_ABANDONED = 0x00000080;
-        public const int SEEK_READ = 0x2;
-        public const int FORWARDS_READ = 0x4;
-        public const int BACKWARDS_READ = 0x8;
-        public const int ERROR_EVENTLOG_FILE_CHANGED = 1503;
-        public const int ERROR_FILE_NOT_FOUND = 2;
+        internal const int ERROR_EVENTLOG_FILE_CHANGED = 1503;
```

Comment:
These can be moved either right next to the API the returns this error (like this: https://github.com/dotnet/corefx/blob/master/src/Common/src/CoreLib/Interop/Windows/Kernel32/Interop.FormatMessage.cs#L17) or to big file all error codes (src\Common\src\CoreLib\Interop\Windows\Interop.Errors.cs). We are using both styles - unclear which one is better.

Reply:
What about to move that to https://github.com/dotnet/corefx/blob/master/src/Common/src/Interop/Windows/kernel32/Interop.FormatMessage_SafeLibraryHandle.cs ?

Reply:
There error codes do not belong to `Interop.FormatMessage_SafeLibraryHandle`.
They are not related in any way to the `FormatMessage` API.

I think just add them to `src\Common\src\Interop\Windows\Interop.Errors.cs` instead.

(The other option would be to add them to `src\Common\src\Interop\Windows\advapi32\Interop.ReadEventLog.cs`.)